types/iva/defects/get/index.d.ts

```diff
@@ -5,21 +5,22 @@
  * and run json-schema-to-typescript to regenerate this file.
  */
 
-export type VehicleType = "psv" | "trl" | "hgv" | "car" | "lgv" | "motorcycle";
 export type InspectionType = "basic" | "normal";
 
 export interface DefectGETIVA {
   sectionNumber: string;
   sectionDescription: string;
-  vehicleTypes: VehicleType[];
   euVehicleCategories: EUVehicleCategory[];
   requiredStandards: {
     rsNumber: number;
     requiredStandard: string;
     refCalculation: string;
     additionalInfo: boolean;
-    inspectionTypes: InspectionType[];
+    inspectionTypes?: InspectionType[];
   }[];
+  additionalInformation?: {
+    notes?: string;
```

Comment:
I don't think we want the notes to be optional. wouldn't we want the outer object to be optional and the inside object to be required

Reply:
Yes I think that sounds like the best option as there won't always be additionalInfo notes recorded